Gemfile.lock

```diff
@@ -59,13 +59,13 @@ GEM
     autoprefixer-rails (4.0.2.2)
       execjs
     awesome_print (1.2.0)
-    aws-sdk (1.59.1)
-      aws-sdk-v1 (= 1.59.1)
-    aws-sdk-v1 (1.59.1)
+    aws-sdk (1.60.2)
```

Comment:
@nik aws-sdk is amazon web services gem for ruby. I have NO idea how it got into gemfile.lock. As you can see it is not in the gemfile. An interdependecy perhaps? Have to check this out.